static/js/brand-store/NotAuthorized/NotAuthorized.test.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import NotAuthorized from "./NotAuthorized";
+
+test("the page should have the correct title", () => {
+  render(<NotAuthorized />);
+
+  expect(screen.getByText("Not authorized")).toBeInTheDocument();
```

Comment:
You could check for heading (even specifying level 1) with getByRole to be more explicit. It is clearly stated in the test description we expect a title.

Reply:
Thanks - done